web/src/main/java/org/akaza/openclinica/control/submit/ListStudySubjectsServlet.java

```diff
@@ -137,7 +137,26 @@ private void createTable() {
         factory.setEventDefintionCRFDAO(getEventDefinitionCRFDAO());
         factory.setStudyGroupDAO(getStudyGroupDAO());
         factory.setStudyParameterValueDAO(getStudyParameterValueDAO());
-        String findSubjectsHtml = factory.createTable(request, response).render();
+       
+        /**
+         * http://localhost:8080/OpenClinica/ListStudySubjects?module=admin&maxRows=25&showMoreLink=true&findSubjects_tr_=true&findSubjects_p_=1&findSubjects_mr_=25
+         */
+       
+        
+        // update session table attributes     
+        HashMap subjectTableSettings = getSubjectTableSettings();
+        
+        String findSubjectsHtml = null;
+        OCRequestWrapper requestWrapper = null;
```

Comment:
why do you need a custom request wrapper?

Reply:
you can't  directly  intercept live request to modify/update request parameters, will  throw   errors, but use custom request wrapper can achieve this